tracer/build/_build/Build.Utilities.cs

```diff
@@ -320,7 +320,7 @@ partial class Build
         .Executes(ReplaceReceivedFilesInSnapshots);
 
     Target PrintSnapshotsDiff  => _ => _
-      .Description("Prints snapshots differences from the current tests")
+      .Description("Prints snapshots differences --compatible-runtimes $compatible_runtimesxfrom the current tests")
```

Comment:
nit: I think this is a typo?
```suggestion
      .Description("Prints snapshots differences from the current tests")
```

Reply:
This is definitely a typo I didn't notice, thank you